src/components/MovieTile/Component/MovieTile.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import { Movie } from "../../Types/movie";
+import styles from "../Styles/MovieTile.module.css";
+interface Props {
+  movie: Movie;
+  onClick: (movie: Movie) => void;
+  onEdit: (movie: Movie) => void;
+  onDelete: (movie: Movie) => void;
+}
+
+export default function MovieTile({ movie, onClick, onEdit, onDelete }: Props) {
+  const { poster_path, title, release_date, genres } = movie;
+  const [isContextMenuOpen, setIsContextMenuOpen] = useState(false);
+
+  const handleContextMenuButtonClick = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.stopPropagation();
+    setIsContextMenuOpen(true);
+  };
+
+  const handleEditClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
+    onEdit(movie);
+    setIsContextMenuOpen(false);
+  };
+
+  const handleDeleteClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
+    onDelete(movie);
+    setIsContextMenuOpen(false);
+  };
+
+  const handleClick = () => {
+    onClick(movie);
+  };
+
+  return (
+    <div className={styles.movieTile} onClick={handleClick}>
+      <div className={styles.poster}>
+        <img src={poster_path} alt={title} />
```

Comment:
React is about UI. Find the way to save the card size and elements position if there is no image. At the moment UI is broken in such case